gradebook/src/GradeBook/book.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+
+/*
+The class Book packages the blueprint book provides ways to:
+1 store the grades. 
+2 add to the given gradebook obj
+3 welcomes the user from args
+4 Assign name to the gradebook
+ */
+
+namespace GradeBook
+{
+    public class book
+    {
+        List<double> grades;
+        string name;
+        double highGrade = double.MinValue;
+        double lowGrade = double.MaxValue;
+        double avg = 0.0;
+
+        //Constructor
+        public book(string name, string[] args) 
```

Comment:
Does this need to take `args`?  For a class like this I would say it shuld only care about its actual parameters (like the user name) and leave argument parsing to the calling code.